Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/censors/dccticketing/DccTicketingJwtCensor.kt

```diff
@@ -0,0 +1,136 @@
+package de.rki.coronawarnapp.bugreporting.censors.dccticketing
+
+import de.rki.coronawarnapp.bugreporting.censors.BugCensor
+import de.rki.coronawarnapp.dccticketing.core.transaction.DccTicketingValidationCondition
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class DccTicketingJwtCensor @Inject constructor() : BugCensor {
+    override suspend fun checkLog(message: String): BugCensor.CensorContainer? {
+        var newMessage = BugCensor.CensorContainer(message)
+        synchronized(jwtSet) {
```

Comment:
this will probably produce problems.
Synchronized is not working with Corountines!

Reply:
true, replaced it a mutex